tools/delete_candidate.php

```diff
@@ -26,264 +41,192 @@
  * @link     https://www.github.com/aces/Loris-Trunk/
  */
 
+// Possible script actions
```

Comment:
@johnsaigle this has come up before, since I was just cleaning up stella's scripts we had decided to keep it as is but considering the amount of changes I've already done... I think this should be considered.

another thing to think about is that if someone is looking to expand this script then the infrastructure is already there, this is sort of an example on how to do it if you want